src/Sentry.Maui/Internal/Disposer.cs

```diff
@@ -0,0 +1,23 @@
+namespace Sentry.Maui.Internal;
+
+// This is a helper we register as a singleton on the service provider.
+// It allows us to register other items that should be disposed when the service provider disposes.
+// TODO: There might be something like this built-in to .NET already.  Investigate and replace if so.
+
+internal class Disposer : IDisposable
```

Comment:
no use case for async  dispose?

Reply:
Not at the moment.  If we need it later we can add it.